lib/main.coffee

```diff
@@ -28,10 +28,12 @@ module.exports =
       lint: (textEditor) =>
         filePath = textEditor.getPath()
         fileText = textEditor.getText()
+        [projectPath] = atom.project.relativizePath(filePath)
+        cwd = if projectPath? then projectPath else path.dirname(filePath)
```

Comment:
Although this is working since Atom exposes `path`, you should have the code explicitly import it.